app/views/organisations/_show_organisation.html.erb

```diff
@@ -23,19 +23,39 @@
   </div>
 <% end %>
 
+<%= render "govuk_publishing_components/components/heading", {
```

Comment:
Maybe a question for @alextea  - should the heading be in a 2/3 column like other headings on GOV.UK?

Reply:
The design shows full width but it can be made 2/3s if needed.

Reply:
It should be in a 2/3 column.